src/libs/httpserver/httpServerProvider.js

```diff
@@ -79,7 +79,8 @@ export const HttpServerProvider = props => {
     }
   }, [path, port])
 
-  const isRunning = () => serverInstance?.isRunning()
+  const isRunning = async () =>
+    (await serverInstance?.isRunning()) ?? false
```

Comment:
What is the purpose of `?? false` here ?

isRunning can return something else than a bool ?

Reply:
isRunning is not called if `serverInstance` in `undefined` so this would return `bool | undefined` without the `?? false`

Reply:
Oh I did not see the `?.` ..